src/apps/mpx.erl

```diff
@@ -58,20 +62,20 @@ handle_client_message(_,_) -> ok.
 
 client_disconnected(Client) -> gen_server:call(?SERVER, {client_disconnected, Client}).
```

Comment:
Line 63 is too long: client_disconnected(Client) -> gen_server:call(?SERVER, {client_disconnected, Client})..